template/src/main.ts

```diff
@@ -14,7 +12,7 @@ if (process.env.ENV === 'development' && module.hot) {
   // first arguments for `module.hot.accept` and `require` methods have to be static strings
   // see https://github.com/webpack/webpack/issues/5668
   makeHot(navbarModuleId, navbarComponent,
-    module.hot.accept('./components/navbar', () => reload(navbarModuleId, (<any>require('./components/navbar')).NavbarComponent)));
+    module.hot.accept(navbarModuleId, () => reload(navbarModuleId, (<any>require(navbarModuleId)).NavbarComponent)));
```

Comment:
Unfortunately, this genuinely has to be left with the static strings. I wish it were otherwise but Webpack won't work with variables. See https://github.com/webpack/webpack/issues/5668#issuecomment-329118902